data/data/powervs/variables-powervs.tf

```diff
@@ -39,6 +39,16 @@ variable "powervs_cloud_instance_id" {
   description = "The cloud instance ID of your account"
 }
 
+variable "powervs_publish_strategy" {
+  type        = string
+  description = "The cluster publishing strategy, either Internal or External"
+  default     = "External"
+  validation {
```

Comment:
Why add validation here? Aren't we checking the value in the install config stage? If we are then this would be another code to maintain if we add more types(though unlikely but possible)

Reply:
Valid point. :) I removed that block